README.md

```diff
@@ -1,23 +1,33 @@
 # ap_verify_ci_hits2015
 
-Data from HiTS (2015) for fast testing of alert production in the LSST DM stack.
+Data from HiTS (2015) for fast testing of alert production in the LSST Science Pipelines.
 
-Contains DECam data from the HiTS (2015) fields `Blind15A_40` and `Blind15A_42`.
+This and other `ap_verify` "datasets" are based on [ap_verify_dataset_template](https://github.com/lsst-dm/ap_verify_dataset_template).
 
+Contains DECam data from the HiTS (2015) fields `Blind15A_40` and `Blind15A_42`:
+
+* Visit 411371, Detectors (ccds) 56 and 60
+* Visit 411420, Detectors (ccds) 5 and 10
+* Visit 419802, Detectors (ccds) 5 and 10
+
+---
+**Note**
 At present, this repository can only be used for Gen 2 processing.
 Support for the new Gen 3 pipeline framework is expected in the future.
```

Comment:
Looking at the [render](https://github.com/lsst/ap_verify_ci_hits2015/blob/d3dd3eb0a70babeefcb60644df147e2cdd3636d8/README.md), the "Note" label prints on the same line as the text. Is that what you intended it to look like?

Reply:
Mostly I wanted the "set apart box" look, don't care much about whether "Note" is in-line or not. I'll play with it.